api.planx.uk/gis/classifiedRoads.js

```diff
@@ -0,0 +1,73 @@
+import "isomorphic-fetch";
+
+// Passport key comes from Digital Planning Schemas googlesheet
+export const PASSPORT_FN = "road.classified";
+
+export const classifiedRoadsSearch = async (req, res, next) => {
+  try {
+    if (!req.query.geom)
+      return next({ status: 401, message: "Missing required query param `?geom=`" });
+
+    // Create an OGC XML filter parameter value which will select the road features (lines)
+    //   that intersect with the buffered site boundary (polygon) coordinates
+    //   ref https://labs.os.uk/public/os-data-hub-examples/os-features-api/wfs-example-intersects#maplibre-gl-js
+    const xml = `
+      <ogc:Filter>
+        <ogc:Intersects>
+        <ogc:PropertyName>SHAPE</ogc:PropertyName>
+          <gml:Polygon srsName="EPSG:4326">
+            <gml:outerBoundaryIs>
+              <gml:LinearRing>
+                <gml:coordinates>${req.query.geom}</gml:coordinates>
+              </gml:LinearRing>
+            </gml:outerBoundaryIs>
+          </gml:Polygon>
+        </ogc:Intersects>
+      </ogc:Filter>
```

Comment:
I was really hoping that we could just buffer here with a `DWITHIN` filter but the OS doesn't support it currently.

From their `GetCapabilities` document - 

```xml
<fes:SpatialOperators>
  <fes:SpatialOperator name="BBOX"/>
  <fes:SpatialOperator name="Equals"/>
  <fes:SpatialOperator name="Disjoint"/>
  <fes:SpatialOperator name="Intersects"/>
  <fes:SpatialOperator name="Touches"/>
  <fes:SpatialOperator name="Crosses"/>
  <fes:SpatialOperator name="Within"/>
  <fes:SpatialOperator name="Contains"/>
  <fes:SpatialOperator name="Overlaps"/>
</fes:SpatialOperators>
```

Reply:
Yep, I had checked this as well (thanks for explicit doc link which I forgot), hence introducing `@turf/buffer` - which still doesn't feel too complicated, but agree `DWITHIN` would have been most ideal here if supported.